yaml/shared/src/main/scala/org/virtuslab/yaml/Position.scala

```diff
@@ -1,9 +1,9 @@
 package org.virtuslab.yaml
 
 final case class Position(offset: Int, line: Int, column: Int, input: Vector[String]):
-  def errorMsg: String =
+  override def toString: String =
```

Comment:
Is it necessary to override the `toString` method here? Maybe, just use errorMsg method name